src/test/scala/mesosphere/mesos/TaskBuilderTest.scala

```diff
@@ -42,9 +42,10 @@ class TaskBuilderTest extends MarathonSpec {
       )
     )
 
-    assert(task.isDefined)
+    assert(task.isInstanceOf[TaskBuilder.BuildSuccess])
 
-    val (taskInfo, taskPorts) = task.get
+    val success = task.asInstanceOf[TaskBuilder.BuildSuccess]
```

Comment:
Avoid `asInstanceOf`.  If you really want to cast and destructure unsafely, do it like this:

```scala
val TaskBuilder.BuildSuccess(taskInfo, taskPorts) = task.get
```